src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/JsonSourceGenerationOptionsTests.cs

```diff
@@ -104,7 +105,8 @@ public static void ContextWithAllOptionsSet_GeneratesExpectedOptions()
             ReadCommentHandling = JsonCommentHandling.Skip,
             UnknownTypeHandling = JsonUnknownTypeHandling.JsonNode,
             UnmappedMemberHandling = JsonUnmappedMemberHandling.Disallow,
-            WriteIndented = true)]
+            WriteIndented = true,
+            IndentText = "\t")]
```

Comment:
Could you also add mixing and matching test cases, e.g. "\f\t "?

Reply:
Here the intent is just to check that the provided value in the `JsonSourceGenerationOptionsAttribute` is correctly passed to the `JsonSerializationOptions`. I do not see the interest to test multiple input values.